docs/statistics.md

```diff
@@ -314,266 +472,230 @@ either already acknowledged or dropped by TSBPD as too late to be delivered.
-Congestion window size, in number of packets. Sender only.
+### pktCongestionWindow
 
-Dynamically limits the maximum number of packets that can be in flight.
-Congestion control module dynamically changes the value.
+The current congestion window size, in packets. Sender only.
 
-In **file mode**  this value starts at 16 and is increased to the number of reported
-acknowledged packets. This value is also updated based on the delivery rate, reported by the receiver.
-It represents the maximum number of packets that can be safely
-sent without causing network congestion. The higher this value is, the faster the
-packets can be sent. In **live mode** this field is not used.
+The congestion window dynamically limits the maximum number of packets that can be "in flight" state. During transmission, congestion control module dynamically changes the size of congestion window.
 
-## pktFlightSize
+In **file mode**, the size of congestion control window is equal to 16 packets at the very beginning and is increased during transmision to the number of reported acknowledged packets. This value is also updated based on the delivery rate reported by the receiver. It represents the maximum number of packets that can be safely sent without causing network congestion. The higher this value is, the faster the packets can be sent. In **live mode**, this field is not used.
+
+### pktFlightSize
 
 The number of packets in flight. Sender only.
 
-`pktFlightSize <= pktFlowWindow` and `pktFlightSize <= pktCongestionWindow`
+The number of packets in flight is calculated as the difference between sequence numbers of the latest acknowledged packet (latest reported by an ACK message packet) and the latest sent packet at the moment statistic is being read. Note that `pktFlightSize <= pktFlowWindow` and `pktFlightSize <= pktCongestionWindow`.
+
+**NOTE:** ACKs are received by the SRT sender periodically at least every 10 milliseconds. This statistic is most accurate just after receiving an ACK packet and becomes a little exaggerated over time until the next ACK packet arrives. This is because with a new packet sent, while the ACK number stays the same for a moment, the value of `pktFlightSize` increases. But the exact number of packets arrived since the last ACK report is unknown. A new statistic might be added to only report the distance between the ACK sequence number and the sent packet sequence number at the moment when an ACK arrives. This statistic will not be updated until the next ACK packet arrives. The difference between the suggested statistic and `pktFlightSize` would then reveal the number of packets with an unknown state at that moment.
+
+### msRTT
+
+The estimation for the round-trip time (RTT), in milliseconds. Available both for sender and receiver.
+
+This value is calculated by the SRT receiver based on the incoming ACKACK control packets (sent back by the SRT sender to acknowledge incoming ACKs).
+
+TODO: peer to agent terminology, with the same journal
```

Comment:
Instead of "peer to agent" etc. you can also say something like "from machine A to machine B and vice versa", though I'm not sure if this speaks better. The journal number is the data that is recorded when ACK message is sent so that it's known what the time was when sending ACK for which the received ACKACK message is a response.